furyctl-on-eks/README.md

```diff
@@ -0,0 +1,522 @@
+
+```bash
+velero backup create --from-schedule manifests test -n kube-system
+```
+
+2. Check the backup status:
+
+```bash
+velero backup get -n kube-system
+```
+
+### (optional) Enforce a Policy with OPA Gatekeeper
+
+OPA Gatekeeper has been deployed as part of the distribution, [the module comes with a set of policies pre-defined][opa-module-docs].
+
+To test drive the default rules, try to create a simple deployment in the `default` namespace:
+
+```bash
+kubectl run --image busybox bad-pod -n default
+```
+
+You should get an error from Gatekeeper saying that the pod is not compliant with the current policies.
+
+Gatekeeper runs as a Validating Admission Webhook, meaning that all the requests to the Kubernetes API server are validated first by Gatekeeper before saving them to the cluster's state.
+
+If you list the pods in the `default` namespace, the list it should be empty, confirming that the pod creation was actually rejected:
+
+```console
+$ kubectl get pods -n default
+No resources found in default namespace.
+```
+
+Some namespaces are exempted from the default policies, for exmaple `kube-system`. Try to create the same pod in the `kube-system` namespace and it should succeed.
+
+```bash
+kubectl run --image busybox bad-pod -n kube-system
+```
+
+Output should be:
+
+```console
+pod/bad-pod created
+```
+
+> 💡 **TIP** Gatekeeper Policy Manger, a simple readonly web UI to easily see the deployed policies and their status is installed as part of the OPA module. You can access it at <http://gpm.internal.demo.example.dev/>
+
+## Step 5 - Teardown
+
+Clean up the demo environment:
+
+1. Delete the EKS cluster and all the related aws resources:
+
+```bash
+furyctl delete cluster
+```
+
+2. Write 'yes' and hit enter, when prompted to confirm the deletion.
+
+3. (Optional) Destroy the S3 bucket holding the Terraform state
+
+```bash
+aws s3api delete-objects --bucket $S3_BUCKET \
+  --delete "$(aws s3api list-object-versions --bucket $S3_BUCKET --query='{Objects: Versions[].{Key:Key,VersionId:VersionId}}')"
+
+aws s3api delete-bucket --bucket $S3_BUCKET
+```
+
+4. Exit from the docker container:
+
+```bash
+exit
+```
+
+## Conclusions
+
+Congratulations, you made it! 🥳🥳
+
+We hope you enjoyed this tour of Fury!
+
+### Issues/Feedback
+
+In case you ran into any problems feel free to [open an issue in GitHub](https://github.com/sighupio/fury-getting-started/issues/new).
+
+### Where to go next?
+
+More tutorials:
+
+- [Fury on GKE][fury-on-gke]
+- [Fury on OVHcloud][fury-on-ovhcloud]
+- [Fury on Minikube][fury-on-minikube]
+
+More about Fury:
+
+- [Fury Documentation][fury-docs]
+
+<!-- Links -->
+[terraform-aws-eks-iam-permissions]: https://github.com/terraform-aws-modules/terraform-aws-eks/blob/v17.24.0/docs/iam-permissions.md
+[fury-getting-started-repository]: https://github.com/sighupio/fury-getting-started/
+[fury-getting-started-dockerfile]: https://github.com/sighupio/fury-getting-started/blob/main/utils/docker/Dockerfile
```

Comment:
this image needs to be updated to include the alpha of the new furyctl

Reply:
how does this work with the other getting started guide that uses furyctl legacy?

Reply:
I propose to use a special tag for the image used in this guide, the other ones don't point to any specific tag (== `latest`) so if we update `latest` we'll break the other guides.